gnocchi/tests/functional/gabbits/base.yaml

```diff
@@ -61,14 +61,14 @@ tests:
   GET: $LOCATION
   status: 200
   response_json_paths:
-      $.archive_policy.name: test1
+      $.archive_policy_name: test1
```

Comment:
This changes the API so that's not good. Is there any reason you can't fetch the archive policy details with the metric and return it like before?

Reply:
Some tests use ``archive_policy_name`` and others ``archive_policy.name`` but I removed the conditional block that chooses between both of them and arbitrarily chose to only keep archive_policy_name  https://github.com/gnocchixyz/gnocchi/pull/436/files#diff-9efde1e460b5fbbfdced1a5cd0b95814L205 . I first tried to replace it with the following block but with the other modifications I made, it is now always hitting the ``else``. I didn't find any way to keep the behavior of the previous conditional block... if you have any clue on that.

    if 'archive_policy' in unloaded:
       d['archive_policy_name'] = self.archive_policy.name
    else:
        d['archive_policy'] = self.archive_policy